Modules/ITS/src/ITSClusterCheck.cxx

```diff
@@ -50,6 +50,17 @@ Quality ITSClusterCheck::check(std::map<std::string, std::shared_ptr<MonitorObje
       }
     }
 
+    if (iter->second->getName().find("LaneStatusSummaryGlobal") != std::string::npos) {
+      auto* h = dynamic_cast<TH1D*>(iter->second->getObject());
+      result.addMetadata("SummaryGlobal", "good");
```

Comment:
Maybe we should change `SummaryGlobal` to `EmptyLaneFractionGlobal` to make code more clear